packaging/linux/deb/host_json.template

```diff
@@ -0,0 +1 @@
+../../../go/kbnm/host_json.template
```

Comment:
These are symlinks, right? Surprisingly, GitHub doesn't show the word "symlink" anywhere in the diff.